### PR TITLE
feat: e2e tests and mobile views

### PR DESCRIPTION
**Pull request by FredrikOseberg**

### CODE REVIEW

File: cypress/integration/feature-toggle/feature.spec.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+                    strategyId = res.body.id;
+                });
+            }
+        ).as('addStrategyToFeature');
+
+        cy.get('[data-test=ADD_NEW_STRATEGY_SAVE_ID]').first().click();
+        cy.get('[data-test=DIALOGUE_CONFIRM_ID]').click();
+        cy.wait('@addStrategyToFeature');
+    });
+});
```
Comment (ivarconr) [inline cypress/integration/feature-toggle/feature.spec.js:245]: nice =)

File: src/component/common/NoItems/NoItems.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        <div className={styles.container}>
+            <div className={styles.textContainer}>{children}</div>
+            <div className={styles.iconContainer}>
+                <NoItemsIcon className={styles.icon} />
+            </div>
+        </div>
+    );
+};
+
+export default NoItems;
```
Comment (ivarconr) [inline src/component/common/NoItems/NoItems.tsx:16]: nice=)

File: src/component/feature/FeatureView2/FeatureStrategies/FeatureStrategiesEnvironments/FeatureStrategiesEnvironmentList/useFeatureStrategiesEnvironmentList.ts [hunk 3]
```diff
@@ -46,7 +48,7 @@ const useFeatureStrategiesEnvironmentList = (strategies: IFeatureStrategy[]) =>
             await updateStrategyOnFeature(
                 projectId,
                 featureId,
-                activeEnvironment.id,
+                activeEnvironment.name,
```
Comment (ivarconr) [inline src/component/feature/FeatureView2/FeatureStrategies/FeatureStrategiesEnvironments/FeatureStrategiesEnvironmentList/useFeatureStrategiesEnvironmentList.ts:51]: good catch!

**Review by ivarconr — COMMENTED** [review]
I saw there where a few mp4 videos (cypress/videos/feature-toggle/feature.spec.js.mp4), they should probably not be checked in?

**Comment by FredrikOseberg** [comment]
> I saw there where a few mp4 videos (cypress/videos/feature-toggle/feature.spec.js.mp4), they should probably not be checked in?

Right. I'll fix!

File: src/component/feature/FeatureView2/FeatureStrategies/FeatureStrategiesEnvironments/FeatureStrategiesConfigure/FeatureStrategiesConfigure.tsx [hunk 4]
```diff
@@ -15,13 +15,16 @@ import { IFeatureViewParams } from '../../../../../../interfaces/params';
 import cloneDeep from 'lodash.clonedeep';
 import FeatureStrategyCreateExecution from '../../FeatureStrategyCreateExecution/FeatureStrategyCreateExecution';
 import { PRODUCTION } from '../../../../../../constants/environmentTypes';
+import { ADD_NEW_STRATEGY_SAVE_ID } from '../../../../../../testIds';
```
Comment (ivarconr) [inline src/component/feature/FeatureView2/FeatureStrategies/FeatureStrategiesEnvironments/FeatureStrategiesConfigure/FeatureStrategiesConfigure.tsx:18]: not important for now;
but i think we can configure our module resolving to support `lib` like this instead:

`import { ADD_NEW_STRATEGY_SAVE_ID } from 'lib/testIds'`;

Comment (ivarconr) [reply src/component/feature/FeatureView2/FeatureStrategies/FeatureStrategiesEnvironments/FeatureStrategiesConfigure/FeatureStrategiesConfigure.tsx:18]: https://www.typescriptlang.org/docs/handbook/module-resolution.html

Comment (ivarconr) [reply src/component/feature/FeatureView2/FeatureStrategies/FeatureStrategiesEnvironments/FeatureStrategiesConfigure/FeatureStrategiesConfigure.tsx:18]: https://decembersoft.com/posts/say-goodbye-to-relative-paths-in-typescript-imports/

**Pull request merged**